src/test/java/com/ecommerce/runners/TestRunner.java

```diff
@@ -8,7 +8,7 @@
 @RunWith(Cucumber.class)
 @CucumberOptions(features = "src/test/java/com/ecommrece/features", 
                  glue = {"com.ecommerce.stepDefinitions", "com.ecommerce.hooks"},
-                 tags = "@test1",
+                 tags = "@addproduct",
 
```

Comment:
Always use one tags key like below 
tags = "@test1 or @addproduct"
update it

Reply:
I already updated